cmd/README.md

```diff
@@ -12,26 +12,32 @@ Command Line tool for managing the WSO2 Micro Integrator
     3. Clone your fork into any directory
     5. `cd` into cloned directory and then cd into `micro-integrator/cmd`
     6. Execute `go mod vendor` or `go mod download` to download all the dependencies
-
-- ### Building
-   `cd` into `micro-integrator/cmd`
-   
-    Execute `./build.sh -t micli.go -v 1.0.0 -f` to build for all platforms. (Can only be built in Mac or Linux)
-      
-    Created packages will be available at `build/target` directory
     
+- ### Building
+    cd into micro-integrator home directory
+
+    Execute `make install` to build both the **Micro Integrator** and **MIcro Integrator CLI** at once.
+
+    Created packages will be available at build/target directory
+
 - ### Running
     Extract the compressed archive generated to a desired location.
     
     Then execute `./micli` to start the application.
     
     Execute `./micli --help` for further instructions.
-    
+
     NOTE: To execute the tool from anywhere, append the location of the executable (micli) to your $PATH variable.
 
 ### Configuration 
-- ### Inbound Listening Port
-    The default listening port of the Management API is `9091`
+
+- ### How to Enable the Management API
+    By default the Management API is disabled. To use the Management API you must use the system property `-DenableManagementInternalAPI` when starting the micro integrator
```

Comment:
This has to be -DenableManagement**Api**  ( enable + <Internal_API_Name> )

Reply:
Fixed in 83b1526